client/settings.js

```diff
@@ -84,85 +84,88 @@ const paymentRequestButtonType = document.getElementById(
 	'woocommerce_woocommerce_payments_payment_request_button_type'
 );
 
-// Payment Request button event listeners.
-paymentRequest.addEventListener( 'change', () => {
-	const inputIds = [
-		'woocommerce_woocommerce_payments_payment_request_button_theme',
-		'woocommerce_woocommerce_payments_payment_request_button_type',
-		'woocommerce_woocommerce_payments_payment_request_button_height',
-	];
-
-	if ( paymentRequest.checked ) {
-		inputIds.forEach( ( id ) => {
+// TODO: Remove this `if` after Apple Pay gets released.
+if ( paymentRequest ) {
```

Comment:
IMO it's better to use `wcpayAdminSettings` for this. E.g. `if (wcpayAdminSettings.paymentRequestEnabled)`.